src/compiler/generate-xspec-tests.xsl

```diff
@@ -49,6 +49,11 @@
             <import href="{$stylesheet-uri}" />
          </xsl:if>
 
+         <xsl:if test="x:helper">
```

Comment:
Should this `<xsl:if>` block go inside the `<xsl:if test="not($is-external)">` block on line 47, because the feature doesn't operate in tests with `run-as="external"`?

Reply:
https://github.com/xspec/xspec/pull/1103#discussion_r461533350